opencti-platform/opencti-front/src/private/components/data/feeds/FeedCreation.jsx

```diff
@@ -671,12 +636,6 @@ const FeedCreation = (props) => {
 
 FeedCreation.propTypes = {
   paginationOptions: PropTypes.object,
-  classes: PropTypes.object,
-  theme: PropTypes.object,
-  t: PropTypes.func,
 };
 
-export default R.compose(
-  inject18n,
-  withStyles(styles, { withTheme: true }),
-)(FeedCreation);
+export default FeedCreation;
```

Comment:
@CelineSebe I pushed a little commit to get rid of withStyle also while we're at it 🙂